app/frontend/src/components/forms/manage/ApiKey.vue

```diff
@@ -29,19 +24,28 @@
             :disabled="!canGenerateKey"
             @click="showConfirmationDialog = true"
           >
-            <span>{{ apiKey ? 'Regenerate' : 'Generate' }} api key</span>
+            <span
+              >{{
+                apiKey
+                  ? $t('trans.apiKey.regenerate')
+                  : $t('trans.apiKey.generate')
+              }}
+              {{ $t('trans.apiKey.apiKey') }}</span
+            >
           </v-btn>
         </v-col>
         <v-col cols="12" sm="5" xl="3">
           <v-text-field
             dense
             flat
             hide-details
-            label="Secret"
+            :label="$t('trans.apiKey.label')"
```

Comment:
It looks like trans.apiKey.label is missing from the translations.
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/84c8763b-9d1c-431b-b2fd-49d421ae28c9)

Reply:
There's also a problem where the API key is always visible for languages other than English and Tagalog.
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/484215ce-9bbc-4020-b15b-3073d76b419b)
![image](https://github.com/bcgov/common-hosted-form-service/assets/35532993/8756e5d7-317d-4608-b67e-1e605410ed8b)

Reply:
I was interpreting this condition "showSecret ? 'text' : 'password'". Meanwhile, I should not be doing that.